proc_nutation.py

```diff
@@ -27,15 +27,18 @@
     s.ift('t2')
     if 'amp' not in s.dimlabels:
         # {{{ centering of data using hermitian function test
+        # {{{ (FROM REVIEW) I still don't understand why this is only done for
+        #                   an amplitude-varied only
         best_shift = hermitian_function_test(s['ph2',0]['ph1',1])
         logger.info(strm("best shift is",best_shift))
         s.setaxis('t2', lambda x: x-best_shift).register_axis({'t2':0})
+        # }}}
```

Comment:
see comment

Reply:
you are right about your correction to my question here "p90-varied only" but the question still stands -- why are you only doing this for a p90-varied, and not for amp-varied?!